aws/resource_aws_ssm_association.go

```diff
@@ -88,6 +89,16 @@ func resourceAwsSsmAssociation() *schema.Resource {
 					},
 				},
 			},
+			"compliance_severity": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
```

Comment:
The new resource documentation notes `CRITICAL` as an acceptable value and the SDK does provide a constant for it as well `AssociationComplianceSeverityCritical`. Can you please add it? Otherwise this pull request is good to go. 👍

Reply:
Yes of Course,

This might arised from the fact that I added the validation function later. But it is fixed now.